sonar-docs/analysis/scan/sonarscanner-for-msbuild.md

```diff
@@ -252,7 +252,9 @@ Some analysis parameters can be set for a single MSBuild project by adding them
 
 **Analyzing languages other than C# and VB**
 
-By default, SonarScanner for .NET will only analyze C# and VB files in your project. To enable the analysis of other types of files, these files must be listed in the MSBuild project file (the `.csproj` or `.vbproj` file).
+For newer SDK-style projects (used by .NET Core, .NET 5, and later), the SonarScanner for .NET will analyze all file types that are supported by the available language plugins unless explicitly excluded. 
+
+For older project types, the scanner will only analyze C# and VB files by default. To enable the analysis of other types of files, these files must be listed in the MSBuild project file (the `.csproj` or `.vbproj` file).
```

Comment:
I wouldn't say "project types" as that has a specific meaning in MSBuild ([see here](https://www.codeproject.com/reference/720512/list-of-visual-studio-project-type-guids)).

`For older-style projects...` would be clearer and consistent with the wording of the first paragraph.

Reply:
Suggested alternative wording:

_For older-style projects, the scanner will only analyze files that are listed in the `.csproj` or `.vbproj` project file. Normally this means that only C# and VB files will be analyzed. To enable the analysis of other types of files, include them in the project file._

Reply:
Thanks for the suggestion, I've updated